pkg/driver/helmdriver.go

```diff
@@ -70,12 +70,13 @@ func (d *helmDriver) Initialize(ctx context.Context, clusterName string) (err er
 }
 
 func (d *helmDriver) Install(ctx context.Context,
-	name string, namespace string, source api.PackageOCISource, values map[string]interface{}) error {
+	name string, namespace string, createNamespace bool, source api.PackageOCISource, values map[string]interface{}) error {
```

Comment:
From a design perspective should we only allow this namespace to be created if targetNamespace is set in the Package Spec? Otherwise we would be installing in what we set as defaultNamespace in the package charts. We don't really have a way to inform the user what that defaultNamespace is now, which could lead them to be confused as to where the package actually installed

Reply:
Maybe a solution here could be to generate a targetNamespace value when the Package spec is generated via the CLI